### ISSUE TITLE
Force refresh option for the JWT token isn't accessible from the other functions calling it.

### ISSUE THREAD

**Issue by ameenalakhras** [Kylmakalle/devicecheck#6]
Starting off, i wanted to admire the project, It helped many of my customers and users having better service.
Great work 👏👏.

Force refresh option for the JWT token isn't accessible from the other functions calling it from the _request or validate_device_token methods etc.

it should inherit the *args and **kwargs from it when it is called  in line 335 in the __init__.py file 

the old code:
```
  def _request(self, url, endpoint, payload, retrying_env=False, *args, **kwargs):
      log.debug(f'Sending request to {url}/{endpoint} with data {payload}')

      result = self._session.post(f"{url}/{endpoint}", json=payload, auth=BearerAuth(self.generate_token()))
```

what should it be:
```
  def _request(self, url, endpoint, payload, retrying_env=False, *args, **kwargs):
      log.debug(f'Sending request to {url}/{endpoint} with data {payload}')

      result = self._session.post(f"{url}/{endpoint}", json=payload, auth=BearerAuth(self.generate_token(*args, **kwargs)))
```

so in the case when using validate_device_token i can call it like this validate_device_token(token=token, force_refresh=True)
and that would validate the device token and force refresh it on every request.

**Comment by Kylmakalle** [comment]
Hey, thank you for using this lib, glad it really helps.

Just curious, what's the purpose of token refresh on each API call?  @ameenalakhras 
It will definitely lead to a bit more processing time due to calls for cryptography backends.

**Comment by ameenalakhras** [comment]
not refreshing the token led to a rejection sometimes (lets say if the request was sent on the last millisecond) but for some reason it happened more than often in some startups I support.

in the "not-FANG / huge company" level a bit more processing doesn't matter much for checks in critical operations but can be turned off in normal requests.

thats why having it as an option tends to give you a lot more freedom in using the lib.

**Comment by Kylmakalle** [comment]
I got you @ameenalakhras. Fix in 1.2.1 should cover your case. `pip install --upgrade devicecheck`

I've used an approach from https://github.com/fastlane/fastlane/pull/19175 and reduced default token validity time to 500 seconds. 

If you still experiencing any issues and sure that something requires an improvement, feel free to reopen this issue.
